src/commonmark/en/content/user/control-data-quality.md

```diff
@@ -131,13 +131,17 @@ internal DHIS 2 messaging system.
 1.  Verify that you have created all the validation rules, validation
     rule groups and validation notifications you need.
 
-2.  Open the **Data Administration** app and click **Scheduling**.
+2.  Open the **Scheduler** app and click the add button in the bottom right corner.
 
-3.  If scheduling is active, click **Stop**.
+3.  Choose a suitable Name for the new job.
 
-4.  In the **Data monitoring** section, select **All daily**.
+4. Select the **Monitoring** Job type using the drop-down menu. 
 
-5.  Click **Start**.
+5. Select a running frequency for the job, i.e. when and how often the job should run.
+
+6. Fill in the **Parameters** section, including the Validation rule groups.
+
+7. Press the **Add job** button to confirm the job creation. For more information on adding jobs, see [Scheduling](https://docs.dhis2.org/master/en/dhis2_user_manual_en/data-administration.html#scheduling).
```

Comment:
change
https://docs.dhis2.org/master/en/dhis2_user_manual_en/data-administration.html#scheduling
to a relative link
data-administration.html#scheduling
to make the link version and locale "agnostic"